### PR TITLE
Views

### PR DESCRIPTION
**Pull request by taldr27**
### In this milestone I:
- Added view files for users and posts.
- Added `erb` code inside the view files to grab data from the database.
- Added counters to each post and user.
- Added basic styling to the views.
- Populated DB with some data.
 
image
image
image
image

gif

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,14 @@
...
+    <%= link_to user_path(user), :class => 'user-button' do %>
+      <div class="user-card">
+         <%= image_tag("user.webp", :class => "user-image", :alt => "user-image") %>
+        <div class="user-data">
+          <h1><%= user.name %></h1>
+          <span>Number of posts: <%= user.posts_counter %></span>
+        </div>
+      </div>
+    <% end %>
+  <% end %>
```
Comment (Munsa1) [inline app/views/users/index.html.erb]: - [x] Kindly follow the given wireframe, so as to position your content as in the given wireframe

Your design
image

Given design
image

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,19 @@
...
+        <div class="post-card">
+          <h2>Post: <%= index + 1 %></h2>
+          <p><%= post.text %></p>
+          <span>Comments: <%= post.comments_counter %></span>
+          <span>Likes: <%= post.likes_counter %></span>
+        </div>
+      <% end %>
+        <%= link_to "See All Posts", post_path_url(@user), :class => 'post-button' %>
+    </div>
+</div>
```
Comment (Munsa1) [inline app/views/users/show.html.erb]: - [x] Kindly align the `see all posts` to the center as in the given template

Your design
image

Given design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @taldr27, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by taldr27** [comment]
@Munsa1 thank you very much for your help and review, I'll fix those requested changes asap.

**Review by codecaiine — APPROVED** [review]
Hi @taldr27 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Great views setup :heavy_check_mark: 
- Great requirement application :heavy_check_mark: 
- No rubocop errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**